README.md

```diff
@@ -404,3 +405,15 @@ Useful commands:
 ### Running tests
 
 `make test`
+
+### Running linter
```

Comment:
hey Alec, can you embed these steps into the Makefile? If you want an example of a fairly sophisticated setup take a look at https://gitlab.cee.redhat.com/service/osd-fleet-manager/-/blob/main/Makefile#L62

Reply:
Will do!  Thanks!